Class-Examples/Python/readFileCollections-example/readingFileCollection.py

```diff
@@ -2,7 +2,7 @@
 # Our resource: The Python os module + a handy code example:
 #  https://www.geeksforgeeks.org/how-to-read-multiple-text-files-from-folder-in-python/
 import spacy
-nlp = spacy.cli.download("en_core_web_md")
+#nlp = spacy.cli.download("en_core_web_md")
```

Comment:
I like how they comment the NLP download line because when testing, it downloads every single time unless you don't allow it to compile.